recording-daemon/packet.c

```diff
@@ -111,29 +113,35 @@ void ssrc_tls_state(ssrc_t *ssrc) {
 			dbg("TLS connection to %s doing handshake",
 				endpoint_print_buf(&tls_send_to_ep));
 			ssrc->tls_fwd_poller.state = PS_HANDSHAKE;
-			if ((ret = SSL_connect(ssrc->ssl)) == 1) {
-				dbg("TLS connection to %s established",
-						endpoint_print_buf(&tls_send_to_ep));
-				ssrc->tls_fwd_poller.state = PS_OPEN;
-				streambuf_writeable(ssrc->tls_fwd_stream);
+			if (!tls_disable) {
```

Comment:
Wouldn't it make sense to set state = PS_OPEN immediately here in the non-TLS case?

Reply:
Thanks, make sense.